examples/barchart.rs

```diff
@@ -19,59 +19,51 @@ use std::{
     time::{Duration, Instant},
 };
 
-use crossterm::{
-    event::{self, DisableMouseCapture, EnableMouseCapture, Event, KeyCode},
-    execute,
-    terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
-};
+use crossterm::event::{self, Event, KeyCode};
+use rand::prelude::*;
```

Comment:
I dislike prelude as it hides what exactly is used and where it comes from.

Using `ratatui::prelude::*` in ratatui examples is not ideal in my opinion as it hides what exactly is used, but it seems like a wanted feature so ok. Using prelude from other crates only increases the confusion what's from where and what is used here.

As I seem to have different ideas about good Rust code I stopped looking/caring about this PR after seeing this. As this PR came up again I post this as a head up.

Reply:
`rand::prelude` is used in their docs (first example in https://docs.rs/rand/latest/rand/)

My take on the use of a prelude in Ratatui is that it removes a bunch of unnecessary boilerplate code and seems to have little to no downside. Can you maybe help me understand what problems you see with using it? The first one you mentioned that there's some confusion about what types are from which libraries, but that's something that any modern IDE or editor with LSP supports. Is there a more concrete example of this that you could share where this has caused problems?

What are the other parts of this that you see as not good rust code?

Reply:
Can we be more explicit here what is actually used of rand? I have used rand in different repos but never used their prelude and didn't notice they have one. Looking at my own code, probably this, but I haven't tested it:

```suggestion
use rand::Rng;
```

Reply:
Rand docs use the prelude - that's their blessed path, so I'd stick with it for this instance.